### DISCUSSION TITLE
Invalid get index 'tags'(on base: 'Dictionary')

### DISCUSSION THREAD

**Discussion by eyeslee-048** [nathanhoad/godot_dialogue_manager#294] [Q&A]
That's some problem while I upgrade the  godot_dialogue_manager version, I can run my demo while I use dialogue_manager low version 
my godot version is 4.1.1
![image](https://github.com/nathanhoad/godot_dialogue_manager/assets/84239762/a1028828-5cc0-4f4e-ad4a-1b353f08d7bb)

**Comment by nathanhoad** [comment]
It looks like Godot's import plugin auto-refreshing doesn't work like it used to. I've updated Dialogue Manager to have a fallback for any missing tags data.
